### PR TITLE
Merge stabilization -> mrtk_dev

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Replacement PR for https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8674 to manually fix merge conflicts.

### CODE REVIEW

**Comment by keveleigh** [comment]
Gonna wait for CI to be fixed before merging

**Review by wiwei — APPROVED** [review]

**Pull request merged**
